.travis.yml

```diff
@@ -7,7 +7,6 @@ branches:
 language: php
 
 php:
-  - 5.6
```

Comment:
ARE YOU KIDDING ME?! AND MY PHP 5.6 OLD SH*T SERVER?! I CAN'T UPGRADE!

REVERT REVERT REVERT!

![image](https://user-images.githubusercontent.com/1698357/32396344-16fae75e-c0e5-11e7-892a-212c8f7d2947.png)